server/src/config/Server.ts

```diff
@@ -47,6 +47,7 @@ export class Server {
 
   private allowCors(req: express.Request, res: express.Response, next: express.NextFunction): void {
     res.header('Access-Control-Allow-Origin', '*');
+    res.header('Access-Control-Allow-Methods', '*');
```

Comment:
This should be 
`res.header('Access-Control-Allow-Methods', 'GET,HEAD,POST,PUT,DELETE');`

Reply:
agreed, specifiying methods according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Methods

`GET, POST, PUT, DELETE, OPTIONS`